cmd/tidb-lightning-ctl/main.go

```diff
@@ -308,6 +314,44 @@ func checkpointDump(ctx context.Context, cfg *config.Config, dumpFolder string)
 	return nil
 }
 
+func getLocalStoringTables(ctx context.Context, cfg *config.Config) (err2 error) {
+	var (
+		tables []string
+	)
+	defer func() {
+		if err2 == nil {
+			fmt.Fprintln(os.Stderr, "These tables are missing intermediate files:", tables)
```

Comment:
Confused about this print, err2 == nil means there is no conflict, why still print this ?

Reply:
`err2 == nil` means that `tables` contains reliable result, no connection error / file error happened. We still need to print `tables` to user

Reply:
Then I think it's better to print a message that indicates no error is better?